app/models/forge/match.rb

```diff
@@ -19,12 +19,14 @@ def forge_id
   end
 
   def get_json_api
-    json_api_url = forge.json_api_url(self)
+    json_api_url = URI(forge.json_api_url(self))
     return {} unless json_api_url
 
-    # rubocop:disable Naming/MemoizedInstanceVariableName
-    @json ||= JSON.parse(URI.parse(json_api_url).open('User-Agent' => 'Ohloh.net client').read)
-    # rubocop:enable Naming/MemoizedInstanceVariableName
+    request = Net::HTTP::Get.new(json_api_url)
+    if ENV['GITHUB_AUTH_TOKEN'] && json_api_url.host == 'api.github.com'
+      request['Authorization'] = "token #{ENV['GITHUB_AUTH_TOKEN']}"
+    end
+    get_result(json_api_url, request)
```

Comment:
Rename `json_api_url` to `json_api_uri`. Use `url` naming for strings and `uri` for **URI** objects.